### PR TITLE
added header branding

### LINKED ISSUE #120: Add OSU/UO branding logos.

### PR DESCRIPTION
**Pull request by mccallum**
fixes #120 

added some graphics files, added adequate branding (responsive and retina friendly) and rough util_links in header

### CODE REVIEW

**Comment by terrellt** [comment]
screen shot 2015-06-23 at 8 43 25 am

Is that what you're seeing?

**Comment by mccallum** [comment]
Yes. That's it. Not perfect yet but a good base.

File: app/assets/stylesheets/main.css.scss [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  -ms-filter: "progid:DXImageTransform.Microsoft.gradient(startColorstr=#99000000, endColorstr=#99000000)";
+
+  color: white;  
+  font-size: .75em;
+  height: 118px;
+  padding-top: 5px;
+  width: 192px;
+  float: left;
+
+  @include respond-to(phone) {height: 80px; width: 96px;}
```
Comment (terrellt) [inline app/assets/stylesheets/main.css.scss]: Is there not built in Bootstrap functions for this?

**Comment by terrellt** [comment]
What's the impetus for the blocks rather than the black header at the top, out of curiosity?

File: app/assets/stylesheets/main.css.scss (hunk 1 above)
Comment (mccallum) [inline app/assets/stylesheets/main.css.scss]: I could do it through Bootstrap with more divs though. This was a little solution I found to have an opacity on a background and full opacity on things inside. I'll change it - not wed to it.

**Comment by mccallum** [comment]
A couple of reasons - the black header at the top in OD1 wasted a lot of space in the middle and the stacked logos integrate more with the stacked OSU and UO flags once you're in one of the shool's sets.

File: app/assets/stylesheets/main.css.scss (hunk 1 above)
Comment (terrellt) [inline app/assets/stylesheets/main.css.scss]: I thought bootstrap had some mixin functions for doing exactly this, using their defined breakpoints. Looking like not?

**Comment by terrellt** [comment]
I think the sudden color switch is throwing me off. How does something like 
screen shot 2015-06-23 at 9 24 50 am look to you? (Generated by dropping a container class)

**Comment by mccallum** [comment]
I'd keep the container class. Users start to lose focus on large monitors if everything is allowed to sprawl to full width. We might go there if it becomes a jam-packed site like Amazon but it isn't yet. 

I think you're right about my @include respond-to usage. It could be @media(max-width: $screen-xs-max){} and ditch the mixin I added. I'll clean it up.

**Comment by terrellt** [comment]
@mccallum Alright. The four blocks of color still feels weird to me, but I'm starting to get used to it. @OregonDigital/owners @OregonDigital/developers @OregonDigital/metadata Thoughts on  ?

**Comment by terrellt** [comment]
Given no comments, I'm going to merge this. We can iterate if there are any comments later.

**Pull request merged**